Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -183,15 +233,43 @@ class IsKilling : Prop :=
   /-- We say a Lie algebra is Killing if its Killing form is non-singular. -/
   killingCompl_top_eq_bot : LieIdeal.killingCompl R L ⊤ = ⊥
 
+namespace IsKilling
+
+variable [IsKilling R L]
+
+attribute [simp] killingCompl_top_eq_bot
```

Comment:
I do wish this to be global but the lemma is a structure field so this is the only way to know to tag it as `simp`. I have moved this statement up to be beside the class definition so that this is clearer.